### PR TITLE
General Heretic Balancing Changes

### PR DESCRIPTION
**Pull request by cowbot92**
# Document the changes in your pull request

Locks heretics to only being able to purchase 1 movement ability - Ash Jaunt, Mind Teleport, or Void Teleport. - If heretics are struggling early game with the removal of multiple movement techniques I will adjust other abilities to compensate, or look at viable ways to allow such an antag to have different types of movement to assist. (If you have ideas please tell me.)

Void spell - Diamond Dust - Greatly reduces the range to 2 tiles outward, still potent enough to lock down an area, but a bit more fair for the opposing side, good for escape.

Void spell - Void Blast - reduced duration of frozen tiles/freeze debuffs by half on targets, was too potent.

Eldritch armor "ominous armor" - Reduced from 50/50/50/50 to 25/25/25/25 base,

Eldritch armor upgrade "garnished ominous armor" - upgraded version of the base armor with 50/50/50/50 armor, and .2 slowdown, requires base armor + a diamond. You get the research to upgrade it with the original armor research.

Living heart - Adjusted maximum targets by 1, granting heretics a bit more leeway when choosing their targets.

# Why is this good for the game?

I'm really trying to get heretic in a good spot before I go adding on more paths- which I fully intend to, up next is a rework of curses/partial ash tree if these changes are well enough received and the early game heretics aren't too hampered.

# Testing

Managed to test everything except the target increase, but that should work as it's just changing 1 variable. (famous last words)

# Changelog

:cl:  

tweak: Adjusts void path spells
tweak: Adjusts ominous armor values
tweak: Adjusts amount of targets you can choose

/:cl:

### CODE REVIEW

File: code/modules/antagonists/eldritch_cult/eldritch_items.dm [hunk 1]
```diff
@@ -282,6 +282,28 @@
...
+	item_state = "eldritch_armor"
+	flags_inv = HIDESHOES|HIDEJUMPSUIT
+	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS
+	allowed = list(/obj/item/gun/magic/hook/sickly_blade, /obj/item/forbidden_book)
+	hoodtype = /obj/item/clothing/head/hooded/cult_hoodie/eldritch
+	armor = list(MELEE = 50, BULLET = 50, LASER = 50, ENERGY = 50, BOMB = 25, BIO = 20, RAD = 0, FIRE = 20, ACID = 20) //now it has a reason to be strong
+	resistance_flags = FIRE_PROOF // ash heretic go brrr
+	slowdown = 0.20
+
+/obj/item/clothing/suit/hooded/cultrobes/eldritch/upgraded/equipped(mob/living/user, slot)
```
Comment (Moltijoe) [inline code/modules/antagonists/eldritch_cult/eldritch_items.dm]: You dont need this a second time

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/eldritch_cult/eldritch_items.dm (hunk 1 above)
Comment (cowbot92) [reply code/modules/antagonists/eldritch_cult/eldritch_items.dm]: this is what I get for coding at 3am, resolved

Comment (cowbot92) [reply code/modules/antagonists/eldritch_cult/eldritch_items.dm]: Ok got rid of it

File: code/modules/antagonists/eldritch_cult/eldritch_items.dm [hunk 2]
```diff
@@ -282,6 +282,19 @@
...
+	desc = "An evolved robe, surging with eldritch power. Strange eyes line the inside."
+	icon_state = "eldritch_armor"
+	item_state = "eldritch_armor"
+	flags_inv = HIDESHOES|HIDEJUMPSUIT
+	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS
+	allowed = list(/obj/item/gun/magic/hook/sickly_blade, /obj/item/forbidden_book)
+	hoodtype = /obj/item/clothing/head/hooded/cult_hoodie/eldritch
+	armor = list(MELEE = 50, BULLET = 50, LASER = 50, ENERGY = 50, BOMB = 25, BIO = 20, RAD = 0, FIRE = 20, ACID = 20) //now it has a reason to be strong
+	resistance_flags = FIRE_PROOF // ash heretic go brrr
+	slowdown = 0.20
```
Comment (Moltijoe) [inline code/modules/antagonists/eldritch_cult/eldritch_items.dm]: c'mon, remove the rest of the redundant lines too

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/eldritch_cult/eldritch_items.dm (hunk 2 above)
Comment (cowbot92) [reply code/modules/antagonists/eldritch_cult/eldritch_items.dm]: legally speaking, it's only redundant if you think it is :) But i've removed it as requested.

**Review by Moltijoe — APPROVED** [review]

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**